indra/newview/llappviewer.h

```diff
@@ -90,9 +90,9 @@ class LLAppViewer : public LLApp
 
     // Application control
     void flushLFSIO(); // waits for lfs transfers to complete
-    void forceQuit(); // Puts the viewer into 'shutting down without error' mode.
+    virtual void forceQuit(); // Puts the viewer into 'shutting down without error' mode.
```

Comment:
Declare these `override`, not `virtual`.

Reply:
Done

Reply:
> Declare these `override`, not `virtual`.

It seems we shouldn't mark some functions as `override`
```
In file included from /Users/runner/work/viewer/viewer/indra/newview/tests/lllogininstance_test.cpp:223:
/Users/runner/work/viewer/viewer/indra/newview/tests/../llappviewer.h:87:18: error: 'init' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    virtual bool init();            // Override to do application initialization
```
Clang requires using the same rule for all overridden functions in the class:
either mark all of them as override or do not use this keyword at all